### PR TITLE
LPD-23390 [Must Add] [P3] Automate LPP-48808

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-23390](https://liferay.atlassian.net/browse/LPD-23390)

### CODE REVIEW

File: modules/test/playwright/pages/object-web/object-action/ActionBuilderPage.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        this.optionNotification = page.frameLocator('iframe').getByRole('option', { name: 'Notification' });
+        this.page = page;
+    }
+
+    async chooseNotificationOption() {
+		await this.inputThenCombo.click();
+        await this.optionNotification.click();
+	}
+
+    async checkNotificationsVisible(notificationsName:string[]) {
```
Comment (cgmonte) [inline modules/test/playwright/pages/object-web/object-action/ActionBuilderPage.ts]: Assertions don't belong to a page object, they belong to the test. A page object is a representation of the page and its functionalities. The action builder page does not provide a way for the user to check if notifications are visible. This is needed and done by the test only, and so it belongs there in the `spec.ts`.

File: modules/test/playwright/pages/object-web/ViewObjectDefinitionsPage.ts [hunk 2]
```diff
@@ -118,4 +118,8 @@ export class ViewObjectDefinitionsPage {
 	async viewInModelBuilder() {
 		this.viewInModelBuilderButton.click();
 	}
+
+	async clickEditObjectDefinitionLink(objectDefinitionName: string) {
```
Comment (cgmonte) [inline modules/test/playwright/pages/object-web/ViewObjectDefinitionsPage.ts:122]: Sort this, it should come after `clickDefaultObjectFolder`.

File: modules/test/playwright/fixtures/objectPagesTest.ts [hunk 3]
```diff
@@ -8,18 +8,38 @@ import {test} from '@playwright/test';
 import {ModalAddObjectDefinitionPage} from '../pages/object-web/ModalAddObjectDefinitionPage';
 import {ModelBuilderPage} from '../pages/object-web/ModelBuilderPage';
 import {ViewObjectDefinitionsPage} from '../pages/object-web/ViewObjectDefinitionsPage';
+import { EditObjectDefinitionPage } from '../pages/object-web/EditObjectDefinitionPage';
+import { SidePanelObjectActionPage } from '../pages/object-web/object-action/SidePanelObjectActionPage';
+import { ViewObjectActionsPage } from '../pages/object-web/object-action/ViewObjectActionsPage';
+import { ActionBuilderPage } from '../pages/object-web/object-action/ActionBuilderPage';
 
 const objectPagesTest = test.extend<{
```
Comment (cgmonte) [inline modules/test/playwright/fixtures/objectPagesTest.ts:16]: Hey, we are mixing too much pages here, I guess. Model Builder pages and Edit Object Definition pages should have their own fixtures. Can you create a new one, like `editObjectDefinitionPagesTest.ts`?

File: modules/test/playwright/tests/object-web/objectActions.spec.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        
+        await editObjectDefinitionPage.openActionsTab();
+            
+        await viewObjectActionsPage.openObjectActionSidePanel();
+
+        await sidePanelObjectActionPage.openActionBuilderTab();
+
+        await actionBuilderPage.chooseNotificationOption();
+
+        await actionBuilderPage.checkNotificationsVisible(names);
```
Comment (cgmonte) [inline modules/test/playwright/tests/object-web/objectActions.spec.ts]: Bring the assertion of `checkNotificationsVisible` here, as mentioned before in another commit comment, assertions belong to the test.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hey @pedro-oliveira446, thanks for working on this! You work is really appreciated! I made some comments and requested some changes. Let me know what you think about them.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by pedro-oliveira446** [comment]
ci:test:playwright

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**